packages/@divvi/mobile/src/navigator/types.tsx

```diff
@@ -299,9 +299,7 @@ export type StackParamList = {
   [Screens.TabDiscover]: {} | undefined
   [Screens.TabHome]: {} | undefined
   [Screens.TabWallet]: { activeAssetTab?: AssetTabType } | undefined
-  [Screens.TabNavigator]: {
-    initialScreen?: Screens.TabHome | Screens.TabWallet | Screens.TabDiscover
-  }
+  [Screens.TabNavigator]: { initialScreen?: string } | undefined
```

Comment:
Maybe we keep the Tabs and also add string, more for documenting and say the string is there for custom tabs defined?